.github/workflows/deploy_combined_service.yml

```diff
@@ -34,26 +38,29 @@ jobs:
       version_to_build: ${{ needs.tag_version.outputs.version_to_tag }}
       owner: ${{ github.repository_owner }}
       application: ${{ github.event.repository.name }}
-  deployment:
-    concurrency: deploy-${{ inputs.environment || 'test' }}
+
+  copilot_deploy:
+    needs: [ tag_version, paketo_build ]
+    strategy:
+      matrix:
+        deployments: [ post-award, post-award-celery, download-report ]
```

Comment:
Just flagging a potential alternative - not saying one way or the other is better. You can decide if you like it or not.

---

If we used the `include` matrix key we could also add a `copilot_command` parameter (or something) to feed the `copilot` command later.

Eg

```suggestion
    strategy:
      matrix:
        include:
          - deployment: post-award
            command: svc
          - deployment: post-award-celery
            command: svc
          - deployment: download-report
            command: job
```

And then at the bottom we could have:

```
      - name: Copilot deploy ${{ matrix.deployments }}
        run: |
          copilot ${{ matrix.command }} deploy --name ${{ matrix.deployments }}
```

Not convinced (or unconvinced) which is better, so you make a call.

Reply:
Oh yeah this is a lot nicer, makes it much more clear and readable. Nice suggestion, will add this now!

Reply:
@samuelhwilliams have added this suggestion to the PR, thanks!